packages/ui/v2/core/Tips.tsx

```diff
@@ -125,3 +137,58 @@ export default function Tips() {
     </div>
   );
 }
+
+export function TipBanner() {
+  const { t } = useLocale();
+  const [animationRef] = useAutoAnimate<HTMLDivElement>();
+
+  const bannerTips = [
+    {
+      id: 1,
+      img: "/team-banner-background-small.jpg",
+      title: t("calcom_is_better_with_team"),
+      subtitle: t("add_your_team_members"),
+      href: `${WEBAPP_URL}/settings/teams/new`,
+      learnMoreHref: "https://go.cal.com/teams-video",
+    },
+  ];
+  const { list, handleRemoveItem } = useLocalList<typeof bannerTips[0]>("removedBannerIds", bannerTips);
+
+  return (
+    <div>
+      {list.map((tip) => {
+        return (
+          <div
+            key={tip.id}
+            ref={animationRef}
+            className="relative -mt-6 mb-6 min-h-[186px] rounded-md"
+            style={{ backgroundImage: "url(" + tip.img + ")", backgroundSize: "cover" }}>
+            <div className="p-6 text-white">
+              <h1 className="font-cal text-lg">{tip.title}</h1>
+              <p className="my-4 max-w-xl text-sm">{tip.subtitle}</p>
+              <div className="space-x-2">
+                <Button className="bg-white" color="secondary" href={tip.href}>
+                  {t("create_team")}
+                </Button>
+                {tip.learnMoreHref && (
+                  <Button
+                    className="!bg-transparent text-white"
```

Comment:
Maybe we should have a comment on why !important is required(which is not recommended)

Reply:
I don't believe there is anything wrong with using !  - The other approach would be to create a whole new component which is not needed for this unique usecase. 

A comment i will add tho :)

Reply:
<img width="827" alt="Screenshot 2022-12-12 at 7 28 25 PM" src="https://user-images.githubusercontent.com/1780212/207063688-57e3a623-6a0d-4d73-8ba1-ada52dbbce3f.png">

Tailwind also mentions that it should be used in rare cases.
But if the purpose is to avoid creating a new component for a unique usecase, that's good enough for me